src/state/transactions/updater.tsx

```diff
@@ -82,12 +82,12 @@ export default function Updater(): null {
       if (!chainId) throw new Error('No library or chainId')
       const retryOptions = RETRY_OPTIONS_BY_CHAIN_ID[chainId] ?? DEFAULT_RETRY_OPTIONS
       return retry(async () => {
-        const res = await fetch(`${GnosisProtocolTrade.getApi(chainId).baseUrl}/api/v1/orders/${orderId}`)
-        if (!res.ok) {
+        const res = await CoWTrade.getCowSdk(chainId).cowApi.getOrder(orderId)
```

Comment:
`res` has type `any`. If possible you can type it

Reply:
Could be due to IntelliSense not picking up the types. The types come from `@cowprotocol/cow-sdk`

<img width="798" alt="image" src="https://user-images.githubusercontent.com/1926216/182580387-3ea27ab2-7e12-4d39-bc73-bd689f16c8ff.png">